test_runner/performance/test_parallel_copy_to.py

```diff
@@ -0,0 +1,140 @@
+from io import BytesIO
+import asyncio
+import asyncpg
+from fixtures.zenith_fixtures import ZenithEnv, Postgres
+from fixtures.log_helper import log
+from fixtures.benchmark_fixture import MetricReport, ZenithBenchmarker
+
+pytest_plugins = ("fixtures.zenith_fixtures", "fixtures.benchmark_fixture")
+
+
+async def repeat_bytes(buf, repetitions: int):
```

Comment:
Nit: doesnt have to be async, might be replaced by [itertools.repeat](https://docs.python.org/3/library/itertools.html#itertools.repeat)